installer/pkg/workflow/destroy.go

```diff
@@ -30,7 +45,39 @@ func destroyTopologyStep(m *metadata) error {
 }
 
 func destroyJoinWorkersStep(m *metadata) error {
-	return runDestroyStep(m, joinWorkersStep)
+	waitTime := time.Duration(3)
+	maxTime := time.Duration(10)
+	for {
```

Comment:
https://godoc.org/k8s.io/apimachinery/pkg/util/wait#Poll

Reply:
and maybe move this wait to `deleteWorkerMachineSet`

Reply:
Done.